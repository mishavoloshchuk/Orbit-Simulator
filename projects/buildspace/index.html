<!DOCTYPE html>
<html lang="ua">
<head>
	<meta charset="UTF-8">
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	<title>Build Space - Будуємо ваше майбутнє!</title>
	<link rel="stylesheet" type="text/css" href="css/style.css">
</head>
<body>
	<div class="toUp" onclick="window.scrollTo(0, 0)">
		Наверх
	</div>
	<header>
		<div class="headerInfoLine">
			<div class="headerInfoLineInner container">
				<ul class="infoFont">
					<li>+(012) 3456789</li>
					<li>Пн - Пт: 8:00 - 17:00 </li>
					<li>info@buildspace.ua</li>
				</ul>
				<ul class="infoFont">
					<li><a href="https://www.YouTube.com">YouTube</a></li>
					<li><a href="https://www.Twitter.com">Twitter</a></li>
					<li><a href="https://www.Facebook.com">Facebook</a></li>
				</ul>	
			</div>
		</div>
		<div class="mainHead">
			<div class="innerMainHead container">
				<a href="index.html"><img src="img/logo.png" alt=""></a>
				<div>
					<h2>пров. Георгіївський, 7, поверх 3</h2>
					<span class="infoFont2">01030, Київ, Україна.</span>	
				</div>
			</div>
		</div>
	</header>
	<section class='sectGallery'>
		<div class="navMenu">
			<div class="navMenuInner container">
				<ul>
					<li onclick="window.location.href = 'aboutcompany.html';">Про компанію</li>
					<li onclick="window.location.href = 'index.html#projects';">Проекти</li>
					<li onclick="window.location.href = 'info.html';">Інформація</li>
					<li onclick="window.location.href = 'cooperation.html';">Співпраця</li>
					<li onclick="window.location.href = 'contacts.html';">Контакти</li>
				</ul>
			</div>
		</div>
		<div class="whiteGradient"></div>
		<div class="imgCollection">
			<img src="http://ibk.ua/wp-content/uploads/revslider/ibk_start/s4.jpg" class="imgDiv">
		</div>
	</section>
	<section class='dblsctt'>
		<div class="backGradient"></div>
		<div class="container dblSect">
			<div class="dblSectItem dblLeftBox">
				<div class="aboutNow">
					<h2>Сьогодні Build Space є лідером на будівельному ринку України</h2>
					<ul>
						<li>Генеральний підрядник у будівництві</li>
						<li>Ремонт та реконструкція будівель</li>
						<li>Оренда спецтехніки та кранів</li>
						<li>Висотне будівництво</li>
					</ul>
				</div>
			</div>
			<div class="dblSectItem ourAchieves">
				<div class="achivItem" style="animation-delay: 0; padding-left: 120px;">
					<img src="img/house3-85x85.png">
					<h2>8 років будуємо</h2>
				</div>
				<div class="achivItem" style="animation-delay: 0.3s; padding-left: 60px;">
					<img src="img/house2-85x85.png">
					<h2>245 завершених проектів</h2>
				</div>
				<div class="achivItem" style="animation-delay: 0.6s;">
					<img src="img/crane-85x85.png">
					<h2>348 800 кв. метрів збудовано</h2>
				</div>
			</div>
		</div>
	</section>
	<section class="weOffer">
		<div class="bgTint"></div>
		<div class="bgGradient"></div>
		<div class="container">
			<h2>Ми пропонуємо </h2><br>
			<h4>Комплексний підхід до будівництва, та беремо на себе повний спектр завдань: від розробки концепції та до здачі об'єкта точно в строк.</h4><br>
			<h2>Наші послуги </h2><br>
		</div>
		<div class="container services">
			<div class="serviceItem" style="transform: translateX(-50vw);">
				<img src="img/2-85x85.png">
				<h3>Технічний замовник</h3>
				<h4>Будівництво будь-якого об'єкта, реставрацію або ремонт неможливо здійснити якісно без генерального проектувальника, який грамотно виконає проект і забезпечить якісне втілення його в реальність.</h4>
			</div>
			<div class="serviceItem" style="transform: translateY(50vw); animation-delay: 0.1s;">
				<img src="img/3-85x85.png">
				<h3>Генеральний проектувальник</h3>
				<h4>Будівництво будь-якого об'єкта, реставрацію або ремонт неможливо здійснити якісно без генерального проектувальника, який грамотно виконає проект і забезпечить якісне втілення його в реальність.</h4>
			</div>
			<div class="serviceItem" style="transform: translateX(50vw);">
				<img src="img/6-85x85.png">
				<h3>Генеральний підрядник</h3>
				<h4>Генпідрядник у будівництві здійснює управління всім процесом зведення об'єкта і налагоджує безперебійну роботу на будмайданчику. ТОВ "ІБК" володіє великим досвідом виконання будівельних робіт, у тому числі в якості генерального підрядника.</h4>
			</div>
		</div>
	</section>
	<section class="doneProjects">
		<div>
			<a name="projects"></a>
			<h2>Виконані проекти </h2><br>
			<div class="projectsImagesBox">
				<div class="width50">
					<div class="width100">
						<div class="width100 divImg" style="background-image: url('img/done_projects/1-1024x576.jpg'); height: 60vh;"></div>
					</div>
					<div class="width100">
						<div class="width50 divImg" style="background-image: url('img/done_projects/e3-1024x540.jpg'); height: 30vh;"></div>
						<div class="width50 divImg" style="background-image: url('img/done_projects/e5-1024x576.jpg'); height: 30vh;"></div>
					</div>
				</div>
				<div class="width50">
					<div  class="width100">
						<div  class="width50">
							<div class="width100 divImg"  style="background-image: url('img/done_projects/e6-1024x576.jpg'); height: 30vh;"></div>
							<div class="width100 divImg"  style="background-image: url('img/done_projects/e7-1024x576.jpg'); height: 30vh;"></div>
						</div>
						<div  class="width50 divImg" style="background-image: url('img/done_projects/e11-576x1024.jpg'); height: 60vh;"></div>
					</div>
					<div class="width100">
						<div class="width50 divImg"  style="background-image: url('img/done_projects/e8-1024x576.jpg'); height: 30vh;"></div>
						<div class="width50 divImg"  style="background-image: url('img/done_projects/s3-1170x600-1024x525.jpg'); height: 30vh;"></div>
					</div>
				</div>
			</div>
		</div>
	</section>
	<footer>
		<div class="footr">
			<div class="container footerInner">
				<img src="img/logo2.png" style="flex-shrink: 0;">
				<div class="footrInfo">
					<div>
						<h3 class="infoFont">Ми працюємо</h3>
						<span class="infoFont2">ПН-ПТ: 8:00-17:00</span><br>
						<span class="infoFont2">СБ-НД: Вихідний</span>
					</div>
					<div>
						<h3 class="infoFont">Контакти</h3>
						<span class="infoFont2">Адрес: 01030, м.Київ, пров Георгіївський, 7, поверх 3.</span><br>
						<span class="infoFont2">Телефон: +380 12 345 6789</span><br>
						<span class="infoFont2"></span><br>
						<span class="infoFont2"></span>
					</div>
				</div>			
			</div>
		</div>
		<div class="footerLine">
			<div class="container infoFont">
				ТОВ "Build Space" ® 2013-2021	
			</div>
		</div>
	</footer>
	<script type="text/javascript" src="javascript/main.js"></script>
</body>
</html>