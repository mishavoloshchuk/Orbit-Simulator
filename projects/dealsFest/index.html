<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<title>Deals Fest - Home</title>
	<link rel="stylesheet" type="text/css" href="css/main.css">
</head>
<body>
	<header class="container">
		<img src="img/Rays.png" style="position: absolute; top: -600px; left: -910px; z-index: -1;">
		<img src="img/Stars effect.png" style="position: absolute; top: 25px; left: -15px; transform: scale(1.2);">
		<h2 class="header_name">Deals Fest</h2>
		<div class="head-menu">
			<div class="checkout">
				<div class="inner-checkout">
					<img src="img/Layer 41.png" alt="" style="margin-left: 10px; cursor: pointer;">
					<span style="font-size: 26px; color: #4D4D4D;">$521</span>
					<div class="checkout_btn" style="font-size: 16px;">Checkout</div>
				</div>			
			</div>
			<div class="authorizing">
				<p>Sigin in   |   Sign up</p>
			</div>
		</div>
	</header>
	<div class="menu container">
		<div style="width: 26.415%; width: 280px; height: 100%; float: left; background: none;"></div>
		<div class="menu_buttons">
			<div class="m_button" style="border-top-left-radius: 7px;">
				Hot Deals
			</div>
			<div class="m_button txd_but" style="
			background: linear-gradient(to bottom, #4587CD, #144C88);
			border: 1px solid #fff6; border-bottom: 1px solid #C9C9C9;">
				<div class="in-txd_but">Deal of the day</div>
			</div>
			<div class="m_button" style="border-top-right-radius: 7px;">
				Most Recent
			</div>
		</div>
		<div class="select_sity">
			Select your sity <span style="font-size: 10px;">▼</span>
		</div>
	</div>
	<div class="container clearfix">
		<div class="left-sidebar">
			<div class="info-box">
				<div class="cost">
					<div class="inner inner-cost">
						<div class="cost_values">
							<div class="cv" style="padding-left: 10px;">
								<div class="cv_txt">Value</div>
								<div class="cv_val">$90</div>
							</div>
							<div class="cv cv_center">
								<div class="cv_txt">Discount</div>
								<div class="cv_val">15.1%</div>	
							</div>
							<div class="cv" style="padding-right: 10px;">
								<div class="cv_txt">Save</div>
								<div class="cv_val">$20</div>	
							</div>
						</div>
					</div>
				</div>
				<div class="time_left">
					<div class="inner inner-time_left">
						<div class="tl_txt">
							Time left to purchase
						</div>
						<div class="tl_tim">
							00 : 08 : 48 : 55
						</div>
					</div>
				</div>
				<div class="time_left_date">
					<div class="inner inner-time_left_date">
						<span style="padding-left: 24px">Days      Hrs      Mints      Secs</span>
					</div>
				</div>
				<div class="save_cost">
					<div class="inner inner-save_cost">$20.00</div>
				</div>
				<div class="buy_now">
					<div class="inner">
						<div class="shortcut"><img src="img/Short glossy 2.png"style="z-index: 2;"></div>
						<div class="buy_now_button">
							<div class="grad_buy_but">
								<div class="inner-buy_now_button">
									Buy Now
								</div>
							</div>
						</div>
					</div>
				</div>
				<div class="recomend">
					<div class="inner inner-recomend">
						<div class="rcmnd_txt">Recommend to friends!</div>
						<div class="soc_net">
							<img src="img/layer 25.png" alt="" style="height: 23px;">
						</div>
					</div>
				</div>
			</div>
			<div class="google_map">
				<div style="overflow:hidden;width: 280px;position: relative;"><iframe width="280" height="280" src="https://maps.google.com/maps?width=280&amp;height=280&amp;hl=en&amp;q=Wuhan%2C%20China+(%D0%9D%D0%B0%D0%B7%D0%B2%D0%B0%D0%BD%D0%B8%D0%B5)&amp;ie=UTF8&amp;t=&amp;z=10&amp;iwloc=B&amp;output=embed" frameborder="0" scrolling="no" marginheight="0" marginwidth="0"></iframe><div style="position: absolute;width: 80%;bottom: 10px;left: 0;right: 0;margin-left: auto;margin-right: auto;color: #000;text-align: center;"></div><style>#gmap_canvas img{max-width:none!important;background:none!important}</style></div>				
			</div>
			<iframe src="https://www.facebook.com/plugins/post.php?href=https%3A%2F%2Fwww.facebook.com%2F20531316728%2Fposts%2F10154009990506729%2F&width=280&show_text=true&height=310&appId" width="280" height="300" style="border:none;overflow:hidden" scrolling="no" frameborder="0" allowTransparency="true" allow="encrypted-media"></iframe>
		</div>
		<div class="content">
			<div class="inner-content">
				<h2>Lorem Ipsum is simply dummy text of the printing</h2>
				<img src="img/Layer 46.png" alt="" style="width: 100%;">
				<br><br>
				<span style="color: #5E5E5E; font-size: 15px;">
					Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries
				</span>
				<br><br>
				<div class="contacts">
					<div style="width: 254px; height: 175px; background-color: #EDEDED; border: 1px solid #D8D4D4; border-radius: 3px; display: inline-block;"></div>
					<div style="width: 47%; height: 100px; padding-left: 20px;">
						<span style="font-size: 17px;">Phone number</span><br>
						<span style="font-size: 26px;">99 - 998 547 533</span>
						<div style="margin-top: 10px;">
							<img src="img/Blue.png" alt="" style="float:left; margin: 10px 10px 0 0;">
							<div style="display: inline; width: 70%;">
								<span style="color: #5E5E5E; font-size: 17px;">
									Address
								</span>
								<br>
								<span style="color: #5E5E5E; font-size: 13px; width: 120px;">
									Lorem Ipsum is simply <br>
									Dummy text of the printing <br>
									And typesetting industry. 
								</span>
							</div>
						</div>
					</div>
				</div>
				<br><br>
				<span style="color: #5E5E5E; font-size: 15px; white-space: pre-wrap;">
Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries but also the leap into electronic typesetting, remaining essentially unchanged.

It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum. release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
				</span>
			</div>
		</div>
		<div class="right-sidebar">
			<div class="inner-right-sidebar">
				<div class="angle">
					<div id="blurred-bg">
					<img src="img/Angle.png">
					</div>
				</div>
				<div><h2 style="padding: 25px 0 0 20px; margin: 0; font-size: 22px; position: relative;">Todays deals</h2></div>
				<div class="event">
					<div style="z-index: 10;"><h3>Lorem Ipsum is simply </h3></div>
					<div class="cost_event">
						<div class="ce">
							<div class="ce_txt">Value</div>
							<div class="ce_val">$90</div>	
						</div>
						<div class="ce" style="border-right: 1px solid #D3D3D3; border-left: 1px solid #D3D3D3; padding: 0 10px;">
							<div class="ce_txt">Discount</div>
							<div class="ce_val">15.1%</div>	
						</div>
						<div class="ce">
							<div class="ce_txt">Save</div>
							<div class="ce_val">$20</div>	
						</div>
					</div>
					<div class="tim_event">
						<div style="width: 93px; height: 100%; background-color: #EDEDED; border: 1px solid #D8D4D4; border-radius: 3px; display: inline-block; z-index: inherit;"></div>
						<div style="padding-left: 15px; z-index: inherit;">
							<img src="img/Time.png" alt=""><br>
							<span style="color: #696969;">Time left</span><br>
							<span style="color: #2E2B2B;">10 : 52 : 12</span>
						</div>
					</div>
					<hr color="#DEDEDE" size="1">
				</div>
				<div class="event">
					<h3>Lorem Ipsum is simply </h3>
					<div class="cost_event">
						<div class="ce">
							<div class="ce_txt">Value</div>
							<div class="ce_val">$90</div>	
						</div>
						<div class="ce" style="border-right: 1px solid #D3D3D3; border-left: 1px solid #D3D3D3; padding: 0 10px;">
							<div class="ce_txt">Discount</div>
							<div class="ce_val">15.1%</div>	
						</div>
						<div class="ce">
							<div class="ce_txt">Save</div>
							<div class="ce_val">$20</div>	
						</div>
					</div>
					<div class="tim_event">
						<div style="width: 93px; height: 100%; background-color: #EDEDED; border: 1px solid #D8D4D4; border-radius: 3px; display: inline-block;"></div>
						<div style="padding-left: 15px;">
							<img src="img/Time.png" alt=""><br>
							<span style="color: #696969;">Time left</span><br>
							<span style="color: #2E2B2B;">10 : 52 : 12</span>
						</div>
					</div>
					<hr color="#DEDEDE" size="1">
				</div>
				<div class="event">
					<h3>Lorem Ipsum is simply </h3>
					<div class="cost_event">
						<div class="ce">
							<div class="ce_txt">Value</div>
							<div class="ce_val">$90</div>	
						</div>
						<div class="ce" style="border-right: 1px solid #D3D3D3; border-left: 1px solid #D3D3D3; padding: 0 10px;">
							<div class="ce_txt">Discount</div>
							<div class="ce_val">15.1%</div>	
						</div>
						<div class="ce">
							<div class="ce_txt">Save</div>
							<div class="ce_val">$20</div>	
						</div>
					</div>
					<div class="tim_event">
						<div style="width: 93px; height: 100%; background-color: #EDEDED; border: 1px solid #D8D4D4; border-radius: 3px; display: inline-block;"></div>
						<div style="padding-left: 15px;">
							<img src="img/Time.png" alt=""><br>
							<span style="color: #696969;">Time left</span><br>
							<span style="color: #2E2B2B;">10 : 52 : 12</span>
						</div>
					</div>
					<hr color="#DEDEDE" size="1">
				</div>
				<div class="event">
					<h3>Lorem Ipsum is simply </h3>
					<div class="cost_event">
						<div class="ce">
							<div class="ce_txt">Value</div>
							<div class="ce_val">$90</div>	
						</div>
						<div class="ce" style="border-right: 1px solid #D3D3D3; border-left: 1px solid #D3D3D3; padding: 0 10px;">
							<div class="ce_txt">Discount</div>
							<div class="ce_val">15.1%</div>	
						</div>
						<div class="ce">
							<div class="ce_txt">Save</div>
							<div class="ce_val">$20</div>	
						</div>
					</div>
					<div class="tim_event">
						<div style="width: 93px; height: 100%; background-color: #EDEDED; border: 1px solid #D8D4D4; border-radius: 3px; display: inline-block;"></div>
						<div style="padding-left: 15px;">
							<img src="img/Time.png" alt=""><br>
							<span style="color: #696969;">Time left</span><br>
							<span style="color: #2E2B2B;">10 : 52 : 12</span>
						</div>
					</div>
					<hr color="#DEDEDE" size="1">
				</div>
				<div class="event">
					<h3>Lorem Ipsum is simply </h3>
					<div class="cost_event">
						<div class="ce">
							<div class="ce_txt">Value</div>
							<div class="ce_val">$90</div>	
						</div>
						<div class="ce" style="border-right: 1px solid #D3D3D3; border-left: 1px solid #D3D3D3; padding: 0 10px;">
							<div class="ce_txt">Discount</div>
							<div class="ce_val">15.1%</div>	
						</div>
						<div class="ce">
							<div class="ce_txt">Save</div>
							<div class="ce_val">$20</div>	
						</div>
					</div>
					<div class="tim_event">
						<div style="width: 93px; height: 100%; background-color: #EDEDED; border: 1px solid #D8D4D4; border-radius: 3px; display: inline-block;"></div>
						<div style="padding-left: 15px;">
							<img src="img/Time.png" alt=""><br>
							<span style="color: #696969;">Time left</span><br>
							<span style="color: #2E2B2B;">10 : 52 : 12</span>
						</div>
					</div>
					<hr color="#DEDEDE" size="1">
				</div>
			</div>
		</div>
	</div>
	<footer class="container">
		<br>
		<span style="font-weight: 100; font-size: 13px;">© Deals Designed by Misha Voloshchuk</span>
		<div class="ft_sotial">
			<img src="img/fb_1.png" alt="Facebook">
			<img src="img/twitter_1.png" alt="Twitter">
			<img src="img/yahoo.png" alt="Yahoo">
			<img src="img/you_tube.png" alt="YouTube">
		</div>
	</footer>
</body>
</html>