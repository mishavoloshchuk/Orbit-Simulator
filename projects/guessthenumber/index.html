<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Відгадай число</title>
    <link rel="stylesheet" href="./styles/styles.css">
</head>
<body>
    <div class="vinett"></div>
    <div class="appOuter">
        <div class="appInner">
            <h2 class="text" style="font-family: 'Comfortaa', cursive; opacity: 0.9;">Відгадай число!</h2>
            <div class="block">
                <label for="maxNum">Максимальне загадуване число:</label><br>
                <input class="input" type="number" min="2" value="100" id="maxNum"><br>
            </div>
            <button class="button" name="smaller">Менше</button>
            <h2 class="text" id="askNum" style="font-weight: bold;">50?</h2>
            <button class="button" name="bigger">Більше</button><br>
            <h2 class="text" id="attempt" style="font-weight: 100;">Спроба №: 1/7</h2><br>
            <button class="button" name="restart">Заново!</button>
        </div>
    </div>
    <div class="backImg"></div>
    <script src="./js/main.js"></script>
</body>
</html>