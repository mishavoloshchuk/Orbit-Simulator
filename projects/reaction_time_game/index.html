<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<title>Время реакции</title>
	<script type="text/javascript" src="js/jquery-3.4.1.js"></script>
	<script type="text/javascript" src="js/script.js"></script>
	<link rel="stylesheet" type="text/css" href="css/style.css">
</head>
<body>
	<div class="screen">
		<div class="text">
			<h1 class="h1" name="Press Ctrl + Click :)">Тест на реакцию</h1>
		</div>
		
	</div>
</body>
</html>