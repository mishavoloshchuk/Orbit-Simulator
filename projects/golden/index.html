<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<title>Document</title>
	<link rel="stylesheet" type="text/css" href="css/main.css">
</head>
<body>
	<header>
		<div class="container">
			<div class="container clearfix">
				<img src="img/logo.png" alt="Golden" class="logo">
				<nav>
					<ul class="menu">
						<li>
							<a href="#">Home</a>
						</li>
						<li>
							<a href="#">Services</a>
						</li>
						<li>
							<a href="#">Portfolio</a>
						</li>
						<li>
							<a href="#">About</a>
						</li>
						<li>
							<a href="#">Contact</a>
						</li>
					</ul>
				</nav>	
			</div>
			<div class="titles">
				<div class="title_first">
					Welcome to our studio!
				</div>
				<h1>
					It's nice to meet tou
				</h1>
			</div>
			<a class="button" href="#">Tell Me More</a>
		</div>
	</header>
	<section id="service_bar">
		<div class="container">
			<div class="title">
				<h2>Services</h2>
				<p>Proin iacolis purus consequat sem cure.</p>
			</div>
			<div class="services clearfix">
				<div class="service_item i1">
					<img src="img/icon1.png" alt="Service">
					<h3>E-Commerce</h3>
					<p>Proin iaculis purus consequant sem cure digne ssim. Dinec porttitore entum suscipit aenean rhoncus posuere odio in tincidunt.</p>
				</div>
				<div class="service_item i2">
					<img src="img/icon2.png" alt="Service">
					<h3>Responsive Web</h3>
					<p>Proin iaculis purus consequant sem cure digne ssim. Dinec porttitore entum suscipit aenean rhoncus posuere odio in tincidunt.</p>
				</div>
				<div class="service_item i3">
					<img src="img/icon3.png" alt="Service">
					<h3>Web Security</h3>
					<p>Proin iaculis purus consequant sem cure digne ssim. Dinec porttitore entum suscipit aenean rhoncus posuere odio in tincidunt.</p>
				</div>
			</div>
		</div>		
	</section>

	<section id="portfolio">
		<div class="container">
			<div class="title">
				<h2>Our portfolio</h2>
				<p>Proin iaculis purus consequat sem cure.</p>
			</div>
			<div class="images clearfix">
				<img src="img/img1.jpg" alt="Image" class="portfolio_img">
				<img src="img/img2.jpg" alt="Image" class="portfolio_img">
				<img src="img/img3.jpg" alt="Image" class="portfolio_img">
			</div>
		</div>		
	</section>

	<section class="about">
		<div class="container">
			<div class="title">
				<h2>About Us</h2>
				<p>Proin iaculis purus consequat sem cure.</p>
			</div>

			<div class="about_item left">
				<div class="title_item item_left">
					<h3 id="about_title_1">Jully 2010</h3>
					<h3 id="about_title_2">Our Huble Beginnings</h3>
					<p>Proin iaculis purus consequant sem cure<br>
						digne ssim. Dinec porttitore entum suscipit<br>
						aenean rhoncus posuere odio in tincidunt.Proin<br>
						iaculis purus consequant sem cure<br>
						digne ssim. Dinec porttitore entum suscipit<br>
					aenean rhoncus posuere odio in tincidunt.</p>
				</div>
				<img src="img/about1.jpg" alt="" class="about_images">
			</div>


			<div class="about_item right">
				<img src="img/about2.jpg" alt="" class="about_images">
				<div class="title_item item_right">
					<h3 id="about_title_1">January 2011</h3>
					<h3 id="about_title_2">Facing Startup Battles</h3>
					<p>Proin iaculis purus consequant sem cure<br>
						digne ssim. Dinec porttitore entum suscipit<br>
						aenean rhoncus posuere odio in tincidunt.Proin<br>
						iaculis purus consequant sem cure<br>
						digne ssim. Dinec porttitore entum suscipit<br>
					aenean rhoncus posuere odio in tincidunt.</p>
				</div>
			</div>


			<div class="about_item left">
				<div class="title_item item_left">
					<h3 id="about_title_1">Decrmber 2012</h3>
					<h3 id="about_title_2">Enter The Dark Days</h3>
					<p>Proin iaculis purus consequant sem cure<br>
						digne ssim. Dinec porttitore entum suscipit<br>
						aenean rhoncus posuere odio in tincidunt.Proin<br>
						iaculis purus consequant sem cure<br>
						digne ssim. Dinec porttitore entum suscipit<br>
					aenean rhoncus posuere odio in tincidunt.</p>
				</div>
				<img src="img/about3.jpg" alt="" class="about_images">
			</div>
			

			<div class="about_item right">
				<img src="img/about4.jpg" alt="" class="about_images">
				<div class="title_item item_right">
					<h3 id="about_title_1">February 2014</h3>
					<h3 id="about_title_2">Our Triumph</h3>
					<p>Proin iaculis purus consequant sem cure<br>
						digne ssim. Dinec porttitore entum suscipit<br>
						aenean rhoncus posuere odio in tincidunt.Proin<br>
						iaculis purus consequant sem cure<br>
						digne ssim. Dinec porttitore entum suscipit<br>
					aenean rhoncus posuere odio in tincidunt.</p>
				</div>


		</div>		
	</section>
	<footer class="footer">
		<p>Â© Copyright 2019 FreebiesXpress.com</p>
	</footer>
</body>
</html>